lua/dashboard/theme/hyper.lua

```diff
@@ -177,10 +177,7 @@ local function map_key(config, key, content)
     local text = content or api.nvim_get_current_line()
     local scol = utils.is_win and text:find('%w') or text:find('%p')
     text = text:sub(scol)
-    local tbl = vim.split(text, '%s', { trimempty = true })
-    local path = tbl[#tbl]
-    path = vim.fs.normalize(path)
-    path = vim.loop.fs_realpath(path)
```

Comment:
normalize just change home dir. you still need fs_realpath

Reply:
I modified the commit. Now it's able to call `config.project.action` correctly. But it still can't deal with that opening a project with spaces in its path. I think it's a bug of telescope.nvim.

```
E5108: Error executing lua: .../packer/opt/dashboard-nvim/lua/dashboard/theme/hyper.lua:184: nvim_exec(): Vim:Error executing Lua callback: .../start/telescope.nvim/lua/telescope/_extensions/init.lua:10: 'find_f
iles' extension doesn't exist or isn't installed: module 'telescope._extensions.find_files' not found:
...
```

Reply:
error mean you don't load the extension of  telescope.

Reply:
there have `fnameescape` function that can use with the filename which have escape character

Reply:
I see. Now it's fixed.